src/main/java/TestBase.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.concurrent.TimeUnit;
+
+public class TestBase {
+    public static WebDriver driver;
+
+    protected String baseUrl;
+
+    public TestBase(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    @BeforeClass
+    public void init() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.get(baseUrl);
+    }
+
+    @AfterClass
```

Comment:
In my opinion you should use Before/After Suite hooks. Use only 1 instance of browser, please.